docs/source/getting_started.md

```diff
@@ -0,0 +1,104 @@
+# Getting started
+
+## Requirements
+
+poliastro requires a number of Python packages, notably:
+
+-   Astropy, for physical units and time handling
+-   NumPy, for basic numerical routines
+-   jplephem, for the planetary ephemerides using SPICE kernels
+-   matplotlib, for static orbit plotting
+-   numba (when using CPython), for accelerating the code
+-   Plotly, for interactive orbit plotting
+-   SciPy, for root finding and numerical propagation
```

Comment:
```suggestion
- Astropy, for physical units and time handling
- NumPy, for basic numerical routines
- jplephem, for the planetary ephemerides using SPICE kernels
- matplotlib, for static orbit plotting
- numba (when using CPython), for accelerating the code
- Plotly, for interactive orbit plotting
- SciPy, for root finding and numerical propagation
```

Reply:
Sure. I think I overlooked it when the conversion was done :sweat_smile: